src/cmd-buildextend-metal

```diff
@@ -43,6 +45,9 @@ while true; do
             build=$2
             shift
             ;;
+        --hostkey)
+            hostkey="/srv/$2"
```

Comment:
this is a tmp solution ( until we have clear answer for "how to share universal key"  )

Reply:
I'm not sure if I follow. Why can't the developer pass in the `/srv/` bit themselves? It seems cleaner to have them pass the full path to the hostkey instead of hardcoding the assumption that it's a path relative to `/srv`.

Reply:
so, `cosa` shell wrapper has this: `-v ${PWD}:/srv/` , and now hostkey is located in `fcos` build dir == `/srv`

Reply:
now `src/config/secex-hostkey` is used by default